code/modules/reagents/chemistry/reagents/pyrotechnic_reagents.dm

```diff
@@ -236,7 +236,7 @@
 
 /datum/reagent/cryostylane/on_mob_add(mob/living/consumer, amount)
 	. = ..()
-	consumer.mob_surgery_speed_mod = 1-((CRYO_SPEED_PREFACTOR * (1 - creation_purity))+CRYO_SPEED_CONSTANT) //10% - 30% slower
+	consumer.mob_surgery_speed_mod = ((CRYO_SPEED_PREFACTOR * (1 - creation_purity))+CRYO_SPEED_CONSTANT) //10% - 30% slower
```

Comment:
undocumented change??

Reply:
this is needed it doesnt change anything